src/core/dep-graph.ts

```diff
@@ -138,6 +138,10 @@ class DepGraphImpl implements types.DepGraphInternal {
     return count;
   }
 
+  public isSame(other: types.DepGraph): boolean {
+    return this.isSameNode(this, 'root-node', other as types.DepGraphInternal, 'root-node');
```

Comment:
please don't assume the root node Id to be `root-node` - it's not enforced. just use `depGraph._rootNodeId` instead 🙏